package.json

```diff
@@ -31,6 +31,7 @@
     "@angular/platform-browser-dynamic": "~13.3.11",
     "@angular/router": "~13.3.11",
     "@briebug/jest-schematic": "^2.1.1",
+    "ajv-formats": "2.1.1",
```

Comment:
this dependency is never used

Reply:
As Netlify said:

![grafik](https://user-images.githubusercontent.com/49990629/196059627-712fae05-eab2-42ed-81ff-3f080ad41cd6.png)